specification/compute/resource-manager/Microsoft.Compute/ComputeRP/stable/2022-08-01/examples/virtualMachineExamples/VirtualMachine_Get_WithDiskControllerType.json

```diff
@@ -0,0 +1,136 @@
+{
+  "parameters": {
+    "subscriptionId": "{subscription-id}",
+    "resourceGroupName": "myResourceGroup",
+    "vmName": "myVM",
+    "api-version": "2022-08-01",
+    "expand": "UserData"
```

Comment:
Update expand param name to $expand as it is defined in the spec and update the value to userData

Reply:
Also can you pls. do the same update for expand param name in the VirtualMachine_Get.json file.

Reply:
will do